drivers/node/node.go

```diff
@@ -102,6 +102,7 @@ type SystemctlOpts struct {
 // BlockDrive provide block drive properties
 type BlockDrive struct {
 	Path       string
+	LABELS     map[string]string
```

Comment:
better to have it in Camel case instead of all Upper caps

Reply:
fixed